pipeline/tasks/gapic_tasks.py

```diff
@@ -61,10 +61,7 @@ def _move_to(self, gapic_config_dir, gapic_api_yaml):
         elif len(gapic_api_yaml) == 0:
             raise ValueError(error_fmt + 'No location specified')
         conf_out = os.path.abspath(gapic_api_yaml[0])
-        if os.path.exists(conf_out):
-            raise ValueError(error_fmt + 'File already exists')
```

Comment:
Yes, I had unstaged changes. It would have made me unhappy if I had accidentally changed artman_pubsub.yaml back to the old path and regenerated on top of my edits. 

You silly people and your aliases. Just do "ctrl+shift+r" and type "comm" for a reverse search through your command-line history, and "git commit" will pop up; likewise for "check" to get "git checkout", or a substring of any command for that matter. Having the full commands in your history is easier to understand and more portable.